packages/context-editor/src/components/AttributePanel.tsx

```diff
@@ -114,6 +118,35 @@ export function AttributePanel({ menuHidden }: { menuHidden: boolean }) {
 		}
 	);
 
+	/** Bulk update version of updateMarkAttr */
+	const updateMarkAttrs = useEditorEventCallback(
```

Comment:
At first I had the menu calling the old `updateMarkAttr` which updates one attr at a time (the same way the original attribute panel did it). However, I realized that now that we have proper forms, we also have proper validation, and so ideally instead of propagating the data directly to the prosemirror editor state, we can validate it first.

for example, the Link popover has a URL field, so we can use our URI validator to make sure it's a proper URL. Then, only once it's valid, do we propagate it to the editor state.

I made the form submit `onBlur` since it's fewer calls than `onChange`, but we could always go back to `onChange` if that's preferred